spec/models/document_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+RSpec.describe Document do
+  let(:collection_ref) { '12345' }
+  let(:attrs) do
+    {
+        collection_ref: collection_ref,
+        name: 'test.doc',
+        last_modified: '2016-12-05T12:20:02.000Z'
+    }
+  end
+  let(:response) { double('Response', body: {collection: '12345', files: [
```

Comment:
I'm late to the party (:grin:) but at some point in the future it would be lovely if the `MojFileUploader` returned some form of domain objects rather than hashes of hashes – but that's for another time!